documentation/adoc/cluster-controller.adoc

```diff
@@ -251,6 +251,7 @@ The configurable fields are the following :
 Default is the namespace where the topic controller is running
 `reconciliationIntervalMs`:: The interval between periodic reconciliations in milliseconds. Default is 900000 (15 minutes).
 `zookeeperSessionTimeoutMs`:: The Zookeeper session timeout in milliseconds. Default is 20000 milliseconds (20 seconds).
+`topicMetadataMaxAttempts`:: The number of attempts for getting topics metadata. Default `4`.
```

Comment:
It says `DEFAULT_TOPIC_METADATA_MAX_ATTEMPTS = 6` in `TopicController`, but then in other places I see 4 floating around. So why are there two defaults, and why are they different?